src/database/init.sql

```diff
@@ -143,15 +146,24 @@ INSERT INTO protocols(id,clinic_id,name,condition) VALUES
     (4,1,'ADHD','ADHD'
 );
 
-INSERT INTO clients (id,passcode,time_passcode,time_passcode_expiry,gov_id,condition,deleted,phone,email,name,gender)
-VALUES
-(1,'M4R70','M4R70', '2022-01-16','211622600','PTSD',false,'0525080784','durd2001@gmail.com','George Joubran', 'male');
-
-INSERT INTO treatment(id,client_id,protocol_id,start_date,status) VALUES
-(1,1,1,'2022-01-16','on-going');
-
-INSERT INTO clients_surveys(id,client_id,survey_id,treatment_id,is_done,is_partially_done,has_missed,survey_snapshot)
+INSERT INTO clients (passcode,time_passcode,time_passcode_expiry,gov_id,condition,deleted,phone,email,name,gender)
 VALUES
-(1,1,1,1,false,false,false,'[{"type":"matrix","title":"Do you feel bothered from:","columns":["Poorly","Semi-Poorly","Avarage","Semi-Strongly","Strongly"],"answers":["0","1","2","3","4"],"instructions":"Below is a list of problems and complaints that people sometimes have in response to stressful life experiences. How much you have been bothered by that problem IN THE LAST MONTH.","questions":[{"id":"1","question":"Feeling very upset when something reminds you of the stressful experience?"},{"id":"2","question":"Trouble remembering important parts of the stressful experience?"},{"id":"3","question":"Loss of interest in activities that you used to enjoy?"},{"id":"4","question":"Irritable behaviour, angry outbursts, or acting aggressively??"}]},{"id":"5","type":"multiple_choice","choice_type":"Radio","question":"Which choice of the choices below you think it will impact you stress the most?","answers":[{"text":"Smoke"},{"text":"Exercide"},{"text":"Drink"},{"text":"Eat"}]},{"id":"6","type":"multiple_choice","choice_type":"Checkbox","question":"Mark the type of injuries you''ve encountered lately:","answers":[{"text":"Physical Pain"},{"text":"Mental Pain"},{"text":"Spiritual Pain"}]},{"type":"open_text","id":"7","question":"Anything else?","placeholder":"Enter your answer here"}]');
+('M4R70','M4R70', '2022-01-16','211622600','PTSD',false,'0525080784','durd2001@gmail.com','George Joubran', 'male');
+
+-- INSERT INTO treatment(id,client_id,protocol_id,start_date,status) VALUES
+-- (1,1,1,'2022-01-16','on-going');
+
+-- INSERT INTO clients_surveys(id,client_id,survey_id,treatment_id,is_done,is_partially_done,has_missed,survey_snapshot)
+-- VALUES

```

Comment:
good rule of thumb, it a comment is not explaining anything in the backend, then just delete it